src/Calculator/Resources/en-US/Resources.resw

```diff
@@ -4106,7 +4106,167 @@
+    <comment>Error that occurs during graping when the equation is too complex to solve.</comment>
+  </data>
+  <data name="EquationHasNoSolution" xml:space="preserve">
+    <value>The equation has no solution.</value>
+    <comment>Error that occurs during graping when the equation has no solution.</comment>
+  </data>
+  <data name="DivideByZero" xml:space="preserve">
+    <value>Cannot divide by zero.</value>
+    <comment>Error that occurs during graping when a divison by zero occurs.</comment>
+  </data>
+  <data name="MutuallyExclusiveConditions" xml:space="preserve">
+    <value>The equation contains logical conditions that are mutually exclusive.</value>
+    <comment>Error that occurs during graping when mutually exclusive conditions are used.</comment>
+  </data>
+    <data name="OutOfDomain" xml:space="preserve">
+    <value>Equation is out of domain.</value>
+    <comment>Error that occurs during graping when the equation is out of domain.</comment>
+  </data>
+    <data name="GE_NotSupported" xml:space="preserve">
+    <value>Graphing this equation is not supported.</value>
+    <comment>Error that occurs during graping when the equation is not supported.</comment>
+  </data>
+  <data name="ParenthesisMismatch" xml:space="preserve">
+    <value>The equation is missing an opening paranthesis.</value>
+    <comment>Error that occurs during graping when the equation is missing a (</comment>
+  </data>
+  <data name="UnmatchedParenthesis" xml:space="preserve">
+    <value>The equation is missing a closing paranthesis.</value>
+    <comment>Error that occurs during graping when the equation is missing a )</comment>
+  </data>
+  <data name="TooManyDecimalPoints" xml:space="preserve">
+    <value>Too many decimal points in a number.</value>
+    <comment>Error that occurs during graping when a number has too many decimals. Ex: 1.2.3</comment>
+  </data>
+  <data name="DecimalPointWithoutDigits" xml:space="preserve">
+    <value>Decimal point is missing digits.</value>
+    <comment></comment>
+  </data>
+  <data name="UnexpectedEndOfExpression" xml:space="preserve">
+    <value>Unexpected end of expression.</value>
+    <comment>Error that occurs during graping when the expression ends unexpectedly. Ex: 3-4*</comment>
+  </data>
+  <data name="UnexpectedToken" xml:space="preserve">
+    <value>Unexpected token in the expression.</value>
+    <comment>Error that occurs during graping when there is an unexpected token.</comment>
+  </data>
+  <data name="InvalidToken" xml:space="preserve">
+    <value>Invalid token in the expression.</value>
+    <comment>Error that occurs during graping when there is an invalid token.</comment>
+  </data>
+  <data name="TooManyEquals" xml:space="preserve">
+    <value>Too many equal signs.</value>
+    <comment>Error that occurs during graping when there are too many equals.</comment>
+  </data>
+  <data name="EqualWithoutGraphVariable" xml:space="preserve">
+    <value>Missing x or y variable.</value>
```

Comment:
"The function must contain at least one x or y variable."

But is this entirely true? I can plot just "6", and that works because the "y=" is implicit. When do we run into this error?

Reply:
Something like 6=1 would throw it iirc

Reply:
Maybe something more generic like "Invalid function" since 6=1 and 6=A are not valid functions.

Reply:
I think when we can be more specific about an error message we should do so, it makes it clearer to the user what exactly is wrong with their input.